server/routes/todo.js

```diff
@@ -10,6 +10,9 @@ router.post("/todo", todoController.todo_post);
 router.get("/todo", todoController.todo_get);
 
 // Create a route to mark item as complete (we could also use this if we wanted to mark the item as not complete)
-router.put("/todo/:id", todoController.todo_complete);
+router.put("/todo/update/:id", todoController.todo_complete);
```

Comment:
I think the previous route was correct.

With REST APIs, the method verb (which in this case would be a "PUT") is descriptive enough of the action you're taking so you shouldn't need to add anything different in the path. The previous path ("/todo/:id") represents the resource on the backend you're trying to access (something in a todos collection with a specific ID).

Reply:
Ok I'll make that change! I'm not on the computer at the moment but I'll update today